datasets/titanic/metadata.json

```diff
@@ -31,10 +43,21 @@
         "description": "Maps gender keys (0, 1) to labeled values.",
         "key": "#{key}",
         "field": [
-            { "name": "key", "@type": "ml:Field", "dataType": "sc:Integer" },
-            { "name": "label", "@type": "ml:Field", "dataType": "sc:String" },
+            {
+              "name": "key",
+              "description": "0 or 1",
```

Comment:
Did you add the description because it was raising an error or a warning?

Reply:
Not specifying a description currently raises a warning.